torchci/lib/drciUtils.ts

```diff
@@ -53,6 +53,12 @@ export const EXCLUDED_FROM_FLAKINESS = [
 // it increases the risk of getting misclassification. This guardrail can
 // be relaxed once we achieve better accuracy from the log classifier. This
 // sets the limit to 7 days
+
+export const ErrorsToNotDisable: RegExp[] = [
```

Comment:
A bit more context: When searching for similar failures, Dr.CI not only compares the capture line, but also the failure context of 12 commands before https://github.com/pytorch/test-infra/blob/main/aws/lambda/log-classifier/src/main.rs#L19.  So, a cancelled job is marked as flaky only when it matches with another exact canceled job from trunk.  I guess it's pretty rate, so we don't see it often till now.